osu.Game/Overlays/Settings/Sections/Input/KeyBindingRow.cs

```diff
@@ -492,6 +501,21 @@ public bool IsBinding
                 }
             }
 
+            private bool isErroring;
+
+            public bool IsErroring
+            {
+                get => isErroring;
+                set
+                {
+                    if (value == isErroring) return;
+
+                    isErroring = value;
+
+                    updateHoverState();
+                }
+            }
```

Comment:
Why does this property exist at all? Making it persistent state like this leads to the following being possible:

https://user-images.githubusercontent.com/20418176/204086187-ec43b477-b0fe-4749-8b94-cf565d5b43c3.mp4

There is zero reason why a binding slot should continue to be red long after the user attempted to use the wrong key.

The visual effect on trying to assign a wrong key should be transient. A brief red flash, a shake animation, etc. work and are idiomatic elsewhere in the game.

Reply:
I can't seem to find any shake animation in-game to base it on.

Reply:
https://github.com/ppy/osu/blob/master/osu.Game/Extensions/DrawableExtensions.cs#L16-L23

Reply:
I cannot make it shake. Maybe because it is in a `FillFlowContainer`.

But I at least made it flash red.